README.md

```diff
@@ -113,16 +113,16 @@ txn.commit();
 
 #### Asynchronous batched operations
 
-You can batch together a set of operations to be processed asynchronously with `node-lmdb`. Committing multiple operations at once can improve performance, and performing a batch of operations and allowing the transaction to sync can be efficiently delegated to an asynchronous thread. The `batchBinary` method accepts an array of write operation requests, where each operation is an array:
```

Comment:
It seems to me that this commit should also be squashed to the previous two. Since it just cleans up stuff that was introduced in there.